### PR TITLE
Request to merge v2 to Master

### PR DESCRIPTION
**Pull request by PhyLimChordata**
The website is now complete and ready to be published. Have a quick lookover and let me know if anything seems broken/needed to be edited.

### CODE REVIEW

File: _site/event/future-leaders-challenge.html [hunk 1]
```diff
@@ -0,0 +1,383 @@
+<!doctype html>
```
Comment (alvintangz) [inline _site/event/future-leaders-challenge.html]: _site folder is in .gitignore, how did these files get in here?

File: index.html [hunk 2]
```diff
@@ -13,7 +13,7 @@
 ---
 
 <div id="homepageCover">
-  <div class="py-5 container px-4 px-md-0">
+  <div class="py-xl-5 hero-container px-4 px-md-0">
```
Comment (alvintangz) [inline index.html]: Didn't do hero here for a reason. Now look at it via mobile

**Comment by alvintangz** [comment]
Please make sure everyone's LinkedIn is up to date

File: index.html [hunk 3]
```diff
@@ -2,78 +2,129 @@
...
+</div>
+<div class="py-5 text-center">
+  <div class="container px-md-5">
+    <h2 class="font-secondary text-primary h1" data-aos="fade-up">Our Events</h2>
+    <div class="row px-lg-5 pt-4">
+      <div class="col-md text-left pr-md-4" data-aos="fade-right">
+        <img src="/assets/img/homepage/flc-thumbnail.png" class="img-fluid" />
+        <p class="mt-4" data-aos="zoom-out-right">Future Leaders Challenge (FLC) is an innovative business case competition designed to give high school students, with a passion for business, the opportunity to face the risk, challenges, and rewards of managing their own companies.</p>
+      </div>
+      <div class="col-md text-left pl-md-4" data-aos="fade-left">
```
Comment (alvintangz) [inline index.html:55]: Padding above this div on mobile please

**Comment by PhyLimChordata** [comment]
Everyone's LinkedIns are up to date. The registration link for CBMs was added.

**Pull request merged**